vllm/envs.py

```diff
@@ -67,6 +67,7 @@
     VERBOSE: bool = False
     VLLM_ALLOW_LONG_MAX_MODEL_LEN: bool = False
     VLLM_RPC_TIMEOUT: int = 10000  # ms
+    VLLM_TIMEOUT_KEEP_ALIVE: int = 5  # seconds
```

Comment:
Let's call it `VLLM_HTTP_TIMEOUT_KEEP_ALIVE` to avoid confusion since there are various other timeouts within vLLM.